### ISSUE TITLE
The app not launching from the menu or quick launcher

### ISSUE THREAD

**Issue by joshting** [unindented/electron-installer-debian#142]
**What version of `electron-installer-debian` are you using?**
0.8.1

**What version of `node` and `npm` are you using?**
npm version 6.0.0

**What operating system are you using?**
Linux Mint 18.3 Cinnamon 64-bit

**Can you compile the [example app](https://github.com/unindented/electron-installer-debian/tree/master/example) successfully?**
Not applicable

**What did you do? Please include the configuration you are using for `electron-installer-debian`.**
electron-installer-debian --src release/builds/linux-x64/ --dest release/installers --arch amd64 --config debian.json

**What did you expect to happen?**
The app should launch from menu or from quick launcher

**What actually happened?**
After the spinner spins for a while, nothing happens.  When I check using the System Monitor, there are three processes running. one with just the app name, second with multiple options related to gpu and stuff and the third with --type=zygote --no-sandbox. If I go to the directory where its installed and run through the command line, it launches fine and there is only one process.

**Comment by malept** [comment]
In order to debug your problem further, we need a minimal testcase to reproduce your problem. Using the [`electron-quick-start`](https://github.com/electron/electron-quick-start) repository as a base, could you please create a minimal Electron app that illustrates the issue you described, and post a link to it here?

**Comment by joshting** [comment]
I tried with electron-quick-start and it does work.  The only difference is app I am working on has and node addon that depends on some libraries on the system.  When I set the application launcher to run from terminal, it threw error on dependencies not found.  So, the conclusion is that when run from command line, it has all the environment variables in place but when running from quick launcher and even with double clicking on the packaged app, the environment setup is not available.

**Comment by joshting** [comment]
Yes, I realized now that its not an electron-installer-debian issue.  The double clicking on the app from the build using electron-packager also does not work.  So, I think setting depends options won't help in this case.  Any suggestions would be appreciated.

**Comment by malept** [comment]
You'll likely need to use [`electron-rebuild`](https://github.com/electron/electron-rebuild). Alternatively, you can switch to Electron Forge (although you'd probably be best served by v6 even though that's in beta) - Forge uses Packager, this module, and `electron-rebuild` under the hood.
